feconf.py

```diff
@@ -183,7 +183,7 @@
 MAX_LEARNER_PLAYLIST_ACTIVITY_COUNT = 10
 
 # The minimum number of training samples required for training a classifier.
-MIN_TOTAL_TRAINING_EXAMPLES = 50
+MIN_TOTAL_TRAINING_EXAMPLES = 10
```

Comment:
Is this change intentional?

Reply:
Oh, this was for testing only. Will revert it.

Reply:
Reverted